webpack/ForemanTasks/Components/TasksTable/TasksBulkActions.js

```diff
@@ -161,3 +168,84 @@ export const bulkCancelById = ({
     }
   }
 };
+
+const handleErrorForceCancel = (error, dispatch) => {
+  dispatch({ type: TASKS_FORCE_CANCEL_FAILURE, error });
+  dispatch(
+    addToast({
+      type: TOAST_TYPES.ERROR,
+      message: `${__(`Cannot force cancel tasks at the moment`)} ${error}`,
+    })
+  );
+};
+
+export const bulkForceCancelById = ({
+  selected,
+  url,
+  parentTaskID,
+}) => async dispatch => {
+  const stopTasks = selected.filter(task => task.state !== 'stopped');
+  if (stopTasks.length < selected.length)
+    dispatch(
+      addToast({
+        type: TOAST_TYPES.WARNING,
+        message: sprintf(
+          '%s task(s) are already stopped',
+          selected.length - stopTasks.length
+        ),
+      })
+    );
```

Comment:
I think this can show up even if there were no already stopped tasks. In that case, warning saying that 0 tasks were already stopped doesn't bring much value

Reply:
selected.length should be more than stoppped.length so it cant be 0

Reply:
Maybe it wasn't from here, but I was getting a warning that 0 tasks are already stopped

Reply:
I'll take a look :)

Reply:
Should be fixed, added `> 0` for `data.skipped_length > 0`